model/atmosphere/dycore/tests/dycore_tests/test_solve_nonhydro.py

```diff
@@ -173,22 +174,15 @@ def test_nonhydro_predictor_step(
         nnow=nnow,
         nnew=nnew,
     )
+    cell_domain = h_grid.domain(dims.CellDim)
+    edge_domain = h_grid.domain(dims.EdgeDim)
 
-    cell_start_lb_plus2 = icon_grid.get_start_index(
-        dims.CellDim, h_grid.HorizontalMarkerIndex.lateral_boundary(dims.CellDim) + 2
-    )
-    cell_start_nudging = icon_grid.get_start_index(
-        dims.CellDim, h_grid.HorizontalMarkerIndex.nudging(dims.CellDim)
-    )
-    edge_start_lb_plus4 = icon_grid.get_start_index(
-        dims.EdgeDim, h_grid.HorizontalMarkerIndex.lateral_boundary(dims.EdgeDim) + 4
-    )
-    edge_start_lb_plus6 = icon_grid.get_start_index(
-        dims.EdgeDim, h_grid.HorizontalMarkerIndex.lateral_boundary(dims.EdgeDim) + 6
-    )
-    edge_start_nuding_plus1 = icon_grid.get_start_index(
-        dims.EdgeDim, h_grid.HorizontalMarkerIndex.nudging(dims.EdgeDim) + 1
-    )
+    cell_start_lb_plus2 = icon_grid.start_index(cell_domain(h_grid.Zone.LATERAL_BOUNDARY_LEVEL_3))
```

Comment:
More consistency renaming (in the tests too ;-) )

```suggestion
    cell_start_lateral_boundary_level_3 = icon_grid.start_index(cell_domain(h_grid.Zone.LATERAL_BOUNDARY_LEVEL_3))
```